src/view/breadcrumbs.component.tsx

```diff
@@ -49,7 +49,7 @@ const Breadcrumbs = (props: BreadcrumbsProps) => {
         <HomeIcon />
       </IconButton>
       <StyledBreadcrumbs
-        separator={<NavigateNext fontSize="small" />}
+        separator={<NavigateNext fontSize="small" aria-label="navigate next" />}
```

Comment:
```suggestion
        separator={<NavigateNext fontSize="small" />}
```

Is this still needed ???